packages/ckeditor5-dev-release-tools/lib/tasks/commitandtag.js

```diff
@@ -6,13 +6,10 @@
 import upath from 'upath';
 import { glob } from 'glob';
 import { simpleGit } from 'simple-git';
-
-const { toUnix } = upath;
-
-const CHUNK_LENGTH_LIMIT = 4000;
+import { git } from '@ckeditor/ckeditor5-dev-utils';
```

Comment:
I'm loudly thinking, because this is quite a big change.

This looks tricky that we have a naming conflict between `simpleGit` and `git`. Maybe we should create a full wrapper for `simple-git` in `@ckeditor/ckeditor5-dev-utils`. This way:
1. We would not have a naming conflict.
2. We would have `simple-git` dependency defined in just one place.
3. We would be able to switch `simple-git` for any other package in the future.

Reply:
I should add the `commit()` function to the `tools` namespace. This will clarify that the function comes from our packages instead of external (`simple-git`).

As for an abstract over Git, this would be a massive topic, as we still have many calls like `tools.shExec( 'git ...' )` in various places in our code. It looks like an unspecified future topic.